test/driver/error_embedding/dune

```diff
@@ -0,0 +1,12 @@
+(executable
+ (public_name raiser)
+ (modules raiser)
+ (libraries ppxlib))
+
+(executable
+ (public_name pp)
+ (modules pp)
+ (libraries ppxlib))
```

Comment:
Nitpick: the two above stanzas can be merged into a single `executables` stanza

Reply:
Nice! I didn't know that stanza.